assets/vue/components/migrate-notice.vue

```diff
@@ -46,30 +46,49 @@
 				dismissed: false,
 			}
 		},
-		computed: {
-			migrationData: function () {
-				return this.$store.state.sitesData.migrate_data
+		props: {
+			migration_data: {
```

Comment:
Variables in JS should be in general named using CamelCase. I know that it's a challenge as I sometimes fail to do this. Also, passing props in Vue is pretty straightforward when it comes to this. You can take a look at the documentation: 

> HTML attribute names are case-insensitive, so browsers will interpret any uppercase characters as lowercase. That means when you’re using in-DOM templates, camelCased prop names need to use their kebab-cased (hyphen-delimited) equivalent[...]

ref: https://vuejs.org/v2/guide/components-props.html

Remember that this is a great place to look for stuff if you're stuck or not sure what is the best practice for something.